testing/unittests/jlibtests.cpp

```diff
@@ -1885,7 +1886,9 @@ element1: !el scalarvalue
                 "cpptestval"
                 "</cpptest>");
         MemoryBuffer mb;
-        mb.reserveTruncate(4*1024+1); // Must be > PTREE_COMPRESS_THRESHOLD (see top of jptree.cpp)
+        byte * data = (byte *)mb.reserveTruncate(4*1024+1); // Must be > PTREE_COMPRESS_THRESHOLD (see top of jptree.cpp)
+        for (unsigned i=0; i < mb.length(); i++)
```

Comment:
Without it was giving spurious failures in debug mode.  (And valgrind complained.)